tools/devtool

```diff
@@ -265,6 +265,20 @@ get_user_confirmation() {
     return 1
 }
 
+# Validate the user supplied version number.
+# It must be composed of 3 groups of integers separated by dot.
+#
+validate_version() {
+    declare version_regex="^([0-9]+.){2}[0-9]+$"
+    version="$1"
+
+    if [ -z "$version" ]; then
+        die "Version cannot be empty."
+    elif [[ ! "$version" =~ $version_regex ]]; then
+        die "Invalid version number: $version".
```

Comment:
An example might help this message. Like "this version you provided is invalid (expecting something like v0.5.1)".